recipes-samples/images/lmp-feature-bluetooth.inc

```diff
@@ -0,0 +1,24 @@
+SRC_URI += "\
```

Comment:
Probably better to rename this to lmp-feature-bt-6lowpan as the only generic bluetooth piece is the part that disables the bluetooth systemd service.